x-pack/plugins/security_solution/public/common/components/landing_page/index.tsx

```diff
@@ -5,11 +5,19 @@
  * 2.0.
  */
 
-import React, { memo } from 'react';
+import React, { memo, useEffect } from 'react';
+import { useSourcererDataView } from '../../containers/sourcerer';
 import { useContractComponents } from '../../hooks/use_contract_component';
+import { updateSourcererData } from '../sourcerer/sourcerer_updater';
 
 export const LandingPageComponent = memo(() => {
   const { GetStarted } = useContractComponents();
+  const { indicesExist } = useSourcererDataView();
+
+  useEffect(() => {
+    updateSourcererData({ indicesExist });
+  }, [indicesExist]);
+
```

Comment:
The `sourcererData$` contract observable is a good idea, but the current implementation is too tied to the Landing page. If this component is never rendered, the `sourcererData$` observable will never be set. Since it is exposed in the plugin contract, the value should be populated consistently and not depend on the page we are rendering. 

If we expose a generic `sourcererData$` observable in the plugin contract, the "offering" plugins should be able to retrieve it and use it on any page. It should be populated consistently and independently of the page we render, containing all the data the sourcerer has, and updating its value when needed.

So, for now, maybe it would be better to simplify, and just pass this kind of values by props like `<GetStarted indicesExist={indicesExist} />? 
This will imply changing the `GetStarted` component type on the serverless and ESS sides, even if it this is not used in ESS. But that's fine.